### DISCUSSION TITLE
Qt 5 support

### DISCUSSION THREAD

**Discussion by louwers** [maplibre/maplibre-native#915] [General]
We are running into an issue with Qt 5 on Windows with the MinGW 8.1 compiler.

Qt 5 is officially [EOL on 26th May 2023](https://endoflife.date/qt).

I want to support it as long as possible (and practical), but I think it also reasonable to slowly move on.

Hence I would like to suggest that we drop support for Qt 5 + MinGW on Windows.

If anyone has a problem with this, please speak up. And in general I would be interested who is using MapLibre GL Native with Qt 5 to be able to tell how long we should still support it.

**Comment by ntadej** [comment]
I am but only with MSVC. In general ANGLE deprecation in Qt6 is not ideal for us for Windows.

Note that Qt5 is so stable that using it now is still completely fine. I agree potentially dropping compilers but I'd not drop full support just yet.

**Reply by louwers** [reply]
That is good to know. And I agree, there is no reason to completely drop support yet.
